generate-docs/scripts/preprocessor.ts

```diff
@@ -7,9 +7,12 @@ import * as fsx from 'fs-extra';
 import yaml = require('js-yaml');
 
 tryCatch(async () => {
+    // ----
+    // Display prompts
+    // ----
     console.log('\n\n');
     const urlToCopyOfficeJsFrom = await promptFromList({
-        message: `What is the source of the d.ts file that should be used to generate the docs?`,
+        message: `What is the source of the office.d.ts file that should be used to generate the docs?`,
```

Comment:
The term "office.d.ts" is slightly confusing since it's named "index.d.ts" on Definitely Typed. How does "Office-js TypeScript definition" sound?

Reply:
Fixed